cibyl/exceptions/model.py

```diff
@@ -31,7 +33,7 @@ def __init__(self, model_type):
 class NoValidSystem(CibylException):
     """Exception for a case when no valid system is found."""
 
-    def __init__(self, valid_systems=None):
+    def __init__(self, valid_systems: List[str] = None):
```

Comment:
For nullable types, it might be best to wrap the types with `Optional` [docs](https://docs.python.org/3/library/typing.html#typing.Optional)